core/util/runChromy.js

```diff
@@ -347,6 +232,8 @@ function processScenarioView (scenario, variantOrScenarioLabelSafe, scenarioLabe
   }
 
   return new Promise((resolve, reject) => {
+    injectBackstopTools(chromy);
```

Comment:
injectBackstopTools() should probably be moved to line 211 -- and only run if `!window.exists`.   Does that work?